docs/markdown/authoritative/performance.md

```diff
@@ -73,11 +73,20 @@ daemon.
 * `signatures`: Number of DNSSEC signatures created
 * `sys-msec`: Number of CPU miliseconds sent in system time
 * `tcp-answers`: Number of answers sent out over TCP
-* `tcp-questions`: Number of questions received over TCP
+* `tcp-answers-bytes`: Total number of answer bytes sent over TCP (since 4.0.0)
+* `tcp4-answers-bytes`: Total number of answer bytes sent over TCPv4 (since 4.0.0)
+* `tcp6-answers-bytes`: Total number of answer bytes sent over TCPv6 (since 4.0.0)
+* `tcp-queries`: Number of questions received over TCP
+* `tcp4-answers`: Number of answers sent out over TCPv4
+* `tcp4-queries`: Number of questions received over TCPv4
+* `tcp6-answers`: Number of answers sent out over TCPv6
+* `tcp6-queries`: Number of questions received over TCPv6
 * `timedout-questions`: Amount of packets that were dropped because they had to wait too long internally
 * `udp-answers`: Number of answers sent out over UDP
-* `udp-queries`: Number of questions received over UDP
```

Comment:
It only moved in these docs. I moved it 4 lines down for some reason, will re-sort just in case. :smile: